src/main/resources/META-INF/plexus/components.xml

```diff
@@ -17,6 +22,7 @@
        <package>org.apache.maven.plugins:maven-jar-plugin:2.6:jar,${project.groupId}:${project.artifactId}:${project.version}:build</package>
        <pre-integration-test>${project.groupId}:${project.artifactId}:${project.version}:start</pre-integration-test>
        <integration-test>org.apache.maven.plugins:maven-failsafe-plugin:2.19.1:integration-test</integration-test>
+       <!-- post-integration-test is opportunity for code coverage, such as jacoco, to collect statistics from running container -->
        <post-integration-test></post-integration-test>
        <verify>${project.groupId}:${project.artifactId}:${project.version}:stop,org.apache.maven.plugins:maven-failsafe-plugin:2.19.1:verify</verify>
```

Comment:
ok, you convinced me ;-). I will put `docker:stop` to *verify* phase. We can still tune this if we get more feedback later on (but I want to make a release today).

The last question is whether `docker:stop` should be before `failsafe:verify` or after ? I guess before, because `failsafe:verify` probably stops the build in case of an error so that `docker:stop` wouldnt run anymore ....

Will revert to you original suggetion.

thanks :)